conan/tools/cmake/file_api.py

```diff
@@ -143,6 +130,7 @@ def to_conan_package(self):
 
             for name, target in self._components.items():
                 component = conan_package.add_component(name)
+                # TODO: CMakeDeps has nothing to do here
```

Comment:
This is important. The model is not and should not be connected at all to CMakeDeps or any other Conan generator. Is a pure model of components, libraries, etc.

Reply:
if it is, how can we store CMake-specific things there?

Reply:
CppPackage should be a generic package information, not connected to CMake not connected to Conan. Conan ``package_info()`` would be the one mapping CppPackage concepts to CMake back via generators.